dvdrip.py

```diff
@@ -331,7 +332,7 @@ def ScanTitle(self, i):
             self.mountpoint], stdout=subprocess.PIPE).split('\n'):
                 if self.verbose:
                         print('< %s' % line.rstrip())
-                yield line
+                yield line.rstrip('\r')
```

Comment:
I think you can instead replace the `.split('\n\)` a few lines above with `.split(os.linesep)`.